pkg/distributor/distributor_test.go

```diff
@@ -1194,16 +1194,16 @@ func TestDistributor_Push_LabelRemoval(t *testing.T) {
 func TestDistributor_Push_ShouldGuaranteeShardingTokenConsistencyOverTheTime(t *testing.T) {
 	ctx := user.InjectOrgID(context.Background(), "user")
 	tests := map[string]struct {
-		inputSeries    labels.Labels
+		inputSeries    []labels.Labels
```

Comment:
I think most of the tests where you made this change should be done the same way `TestDistributor_MetricsForLabelMatchers` was, to minimise diffs and reviewer effort.

Reply:
Done, you are completely right.